src/Compilers/Core/MSBuildTask/Csc.cs

```diff
@@ -241,6 +247,11 @@ protected internal override void AddResponseFileCommands(CommandLineBuilderExten
                 }
             }
 
+            if (HostObject is ICscHostObject6 csHost6)
```

Comment:
We don't need new `ICscHostObject` interfaces anymore. We just passs everything new through the command line arguments parameter.